localization/languages/pt-BR.json

```diff
@@ -31,89 +31,89 @@
         "defaultTaskName": "Tarefa %n", //this is the name used for newly-created tasks; %n is replaced with a number ("task 1", "task 2", etc)
-        "taskDeleteWarning": {"unsafeHTML": null}, //missing translation
-        "returnToTask": null, //missing translation
-        "taskDescriptionTwo": null, //missing translation
-        "taskDescriptionThree": null, //missing translation
+        "taskDeleteWarning": {"unsafeHTML":"Tarefa deletada! <a> desfazer?</a>"},
+        "returnToTask": "Retornar a sua tarefa anterior",
+        "taskDescriptionTwo":"%t e %t",
+        "taskDescriptionThree":"%t e %t, mais %n", 
         /* find in page toolbar */
         "searchInPage": "Buscar na página", //this is used as the placeholder text for the textbox in the find in page toolbar
         "findMatchesSingular": "Ocorrência %i de %t", //this and the next label are used to indicate which match is currently highlighted
         "findMatchesPlural": "Ocorrência %i de %t",
         /* Focus mode */
         "isFocusMode": "Você está no modo concentração.",
         "closeDialog": "OK", //used as a label for the button that closes the dialog
-        "focusModeExplanation1": "No modo concentração, todas as abas com exceção da atual ficam escondidas e você não pode criar novas abas.",
+        "focusModeExplanation1": "No modo concentração, todas as guias com exceção da atual ficam escondidas e você não pode criar novas abas.",
         "focusModeExplanation2": "Você pode desativar o modo concentração desmarcando a caixa \"modo concentração\" no menu visualizar.",
         /* relative dates */
-        "timeRangeJustNow": null, //missing translation
-        "timeRangeMinutes": null, //missing translation
-        "timeRangeHour": null, //missing translation
-        "timeRangeToday": null, //missing translation
-        "timeRangeYesterday": null, //missing translation
-        "timeRangeWeek": null, //missing translation
-        "timeRangeMonth": null, //missing translation
-        "timeRangeYear": null, //missing translation
-        "timeRangeLongerAgo": null, //missing translation
+        "timeRangeJustNow":"Agora mesmo",
+        "timeRangeMinutes": "Há alguns minutos atrás",
+        "timeRangeHour":"Na última hora",
+        "timeRangeToday":"Hoje",
+        "timeRangeYesterday":"Ontem",
+        "timeRangeWeek": "Na semana passada",
+        "timeRangeMonth":"No mês passado",
+        "timeRangeYear":"No ano passado",
+        "timeRangeLongerAgo":"Há mais tempo",
         /* pages/error/index.html */
         "crashErrorTitle": "Ocorreu algo de errado.",
-        "crashErrorSubtitle": "Ocorreu um problema ao tentar mostrar esta página.",
+        "crashErrorSubtitle": "Erro ao carregar página.",
         "errorPagePrimaryAction": "Tentar novamente",
         "serverNotFoundTitle": "Servidor não encontrado",
-        "serverNotFoundSubtitle": "Min não conseguiu achar este site.",
+        "serverNotFoundSubtitle": "Site não localizado.",
```

Comment:
The original translation seems more correct to me here, since the [English version](https://github.com/minbrowser/min/blob/master/localization/languages/en-US.json#L95) is "Min couldn't find this website".